src/cache/diagnostic_edmf_precomputed_quantities.jl

```diff
@@ -365,6 +382,18 @@ function set_diagnostic_edmf_precomputed_quantities!(Y, p, t)
                         entr_detrʲ_prev_level.entr * u³ʲ_data_prev_halflevel
                     )
                 )
+
+            @. u³ʲ_datau³ʲ_data -=
+                (
+                    1 /
+                    (local_geometry_halflevel.J * local_geometry_halflevel.J)
+                ) * (
+                    local_geometry_prev_level.J *
+                    local_geometry_prev_level.J *
+                    2 *
+                    nh_pressureʲ_data_prev_level
+                )
+
```

Comment:
What is the scale_factor in the next line?

Reply:
That's not a good name. It's the threshold for setting everything to grid mean.

Reply:
I changed it to `minimum_value`.